src/main/java/com/epam/tc/hw7/entities/MetalsColors.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.tc.hw7.entities;
+
+import com.epam.jdi.tools.DataClass;
+import java.util.List;
+
+public class MetalsColors extends DataClass<MetalsColors> {
+    public String oddsSelector;
```

Comment:
odd and even are numbers

Reply:
Yes they are numbers, but their format is string and much easier to read them as strings

Reply:
Ok, somebody write 'a' as a value to your test data

Reply:
Because of the format itself on the site, if we use the int format for select(), there will be an error, since '5' or '7', for example, are string values, and we have  only 4 selectable values. Thinking that someone can write 'a' in our JSON, then it will already be an error in the provided data, since in this way the names may not be transmitted correctly, with errors, etc.